creator-node/src/fileManager.js

```diff
@@ -41,16 +42,24 @@ async function saveFileFromBufferToIPFSAndDisk (req, buffer) {
  *
  * @dev - only call this function when file is already stored to disk, else use saveFileFromBufferToIPFSAndDisk()
  */
-async function saveFileToIPFSFromFS (req, srcPath) {
+async function saveFileToIPFSFromFS ({ logContext }, req, srcPath, ipfs) {
```

Comment:
Can we explicitly pass in cnodeUserUUID and remove the req param?

Reply:
done